inspirehep/modules/pidstore/utils.py

```diff
@@ -45,6 +45,29 @@ def get_endpoint_from_pid_type(pid_type):
     return PID_TYPE_TO_ENDPOINT[pid_type]
 
 
+def get_endpoint_from_schema(schema):
```

Comment:
I know, this sucks. The problem is that we have so many concepts that are tightly related but are all expressed by strings with subtle differences, and conversion functions are the cleanest interface I could think of.